orchestra/project_api/tests/test_project_api.py

```diff
@@ -575,12 +587,22 @@ def test_get_list_of_todos_with_filters(self):
         self.assertEqual(len(resp.json()), 1)
         self.assertEqual(resp.json()[0]['step'], self.todo_with_step.step.slug)
 
-        url_with_filters = '{}?project={}&step__slug={}'.format(
+        url_with_filters = '{}?project__id={}&step__slug={}'.format(
             self.list_url, self.project.id, self.todo_with_step.step.slug)
         resp = self.request_client.get(url_with_filters)
         self.assertEqual(resp.status_code, 200)
         self.assertEqual(resp.json()[0]['step'], self.todo_with_step.step.slug)
 
+        ids_to_filter_by = [self.todo.id, self.todo_with_step.id]
```

Comment:
What if you pass an id that does not exist? Having that test might be useful! :)

Reply:
Done :)